### PR TITLE
Expect changes on acceptance test.

### LINKED ISSUE #123: Cron fails with diff in default character set
The cron GHA is currently failing with a difference in the default character set for the created DB.

Probably need to explicitly specify the default character set.

### PR DESCRIPTION
**Pull request by jkodroff**
Fixes #123

### CODE REVIEW

File: examples/examples_nodejs_test.go [hunk 1]
```diff
@@ -14,6 +14,10 @@ func TestAccDatabase(t *testing.T) {
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "database"),
+			// The default character set for the MySQL Docker image has diverged from the upstream provider's value.
+			// Since the upstream provider has been archived and will not be updated, we'll simply expect changes after
+			// creating a DB with the default options:
+			ExpectRefreshChanges: true,
```
Comment (stack72) [inline examples/examples_nodejs_test.go:20]: Needs to also add `AllowEmptyPreviewChanges: true`

**Comment by jkodroff** [comment]
@stack72 Still failing, apparently in the same way. Any suggestions?

File: examples/examples_nodejs_test.go [hunk 2]
```diff
@@ -14,6 +15,12 @@ func TestAccDatabase(t *testing.T) {
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "database"),
+			// The default character set for the MySQL Docker image has diverged from the upstream provider's value.
+			// Since the upstream provider has been archived and will not be updated, we'll simply expect changes after
+			// creating a DB with the default options:
+			AllowEmptyPreviewChanges: true,
+			AllowEmptyUpdateChanges:  true,
+			ExpectRefreshChanges:     true,
```
Comment (guineveresaenger) [inline examples/examples_nodejs_test.go:23]: this is more of a question than a comment 😂 but I presume it is not possible/advisable to recreate the *test* resources to match the state of the provider at archive?

**Review by guineveresaenger — APPROVED** [review]

File: examples/examples_nodejs_test.go (hunk 2 above)
Comment (stack72) [reply examples/examples_nodejs_test.go:23]: > this is more of a question than a comment 😂 but I presume it is not possible/advisable to recreate the *test* resources to match the state of the provider at archive?

This test result is *nothing* to do with the archived provider. It's due to a change in the upstream container

**Pull request merged**